### ISSUE TITLE
[FEATURE] Hide if chat is in sidebar

### ISSUE THREAD

**Issue by fabiocmg** [Moerill/fvtt-chat-notifications#19]
Would be good if the notification was not shown if the chat was showing in the sidebar.

**Comment by kiligir** [comment]
Yeah I'm just going to sneak on in here and suggest that also. In fact, the module is somewhat annoying without this feature.
